deploy/basic/terraform/policies.tf

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2019, 2020 Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# 
+
+# Create lifecycle policy to delete temp files
+resource "oci_objectstorage_object_lifecycle_policy" "mushop_deploy_assets_lifecycle_policy" {
```

Comment:
IIRC this was causing problems for us initially. Has this been verified?

Reply:
yes. Is being correctly generated and on the right place. If someone has access to a specific compartment and not to the tenancy, now works fine.